bundles/org.openhab.binding.network/src/main/resources/OH-INF/i18n/network.properties

```diff
@@ -88,3 +92,8 @@ channel-type.network.rateDown.label = Download Rate
 channel-type.network.rateDown.description = Current download rate
 channel-type.network.rateUp.label = Upload Rate
 channel-type.network.rateUp.description = Current upload rate
+
+# thing types config
+
+thing-type.config.network.pingdevice.usePing.label = Use ICMP Ping
+thing-type.config.network.pingdevice.usePing.description = Set to true if the presence detection is allowed to use icmp ping. If you are monitoring network latency using arping, you should switch this off to prevent mixing results with arp ping results.
```

Comment:
```suggestion
```

Reply:
Automatic generation left it in there?

Reply:
Yes, the tool will not delete anything, but move unused strings down to a duplicate section. It's probably on purpose so it will be caught when reviewing the changes in case something is actually used (dynamically) without the tool detecting it. In any case, it requires manual removal.